tests/kafkatest/services/connect.py

```diff
@@ -57,6 +57,7 @@ def __init__(self, context, num_nodes, kafka, files):
         self.kafka = kafka
         self.security_config = kafka.security_config.client_config()
         self.files = files
+        self.kafka_heap_opts = None
```

Comment:
Should this just be more generic? Seems like we could end up adding flags for a ton of different envvars. Is there any issue with having the caller set the envvars more directly since they have to know what the available settings are anyway?